### ISSUE TITLE
what is the difference between stability.trim.contigs.good.fasta and stability.trim.contigs.fasta and the same for summary file 

### ISSUE THREAD

**Issue by ManarRashad** [mothur/mothur#725]
while using 
mothur > summary.seqs(fasta=stability.trim.contigs.fasta)
this gets 152360 sequences and other details which conclude that the bps of sequences ranged from  248 and 253 bases.
So I just need what is the need for running 
mothur > screen.seqs(fasta=stability.trim.contigs.fasta, group=stability.contigs.groups, maxambig=0, maxlength=275)
this gets 128872 sequences and other details also that I couldn't understand
and why exactly maxlength = 275 

in summary 
1- why we need to make summary.seqs and after it screen.seqs ?
2- what is the difference between their results?
3- why we use here 275 ?

**Comment by mothur-westcott** [comment]
1- why we need to make summary.seqs and after it screen.seqs ?

The summary.seqs command is not required. It is designed to provide a brief summary of your dataset.

2- what is the difference between their results?

The screen.seqs command is designed to remove sequences from your dataset that fail to meet the requirements you specify. In the command above, any read longer than 275 bps or with ambiguous bases will be removed.

3- why we use here 275 ?

Paired reads are 250 bp long. 275 ensures good overlap in the pairing of the reads.

A few things you way find helpful as a new mothur user:

https://github.com/mothur/mothur/issues/216 - new user links
https://mothur.org/wiki/summary.seqs/ - wiki page for summary.seqs
https://mothur.org/wiki/screen.seqs/ - wiki page for screen.seqs

https://aem.asm.org/content/79/17/5112
https://pubmed.ncbi.nlm.nih.gov/22194782/
